Sources/Apodini/Semantic Model Builder/gRPC/GRPCExporterUtility.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  GRPCExporterUtility.swift
```

Comment:
I prefer a different naming convention for extensions: `Endpoint+GRPC.swift`

Utility is very ambiguous and the former has a long history 😅 

> A category is usually declared in a separate header file and implemented in a separate source code file. In the case of XYZPerson, you might declare the category in a header file called XYZPerson+XYZPersonNameDisplayAdditions.h.

From: https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/CustomizingExistingClasses/CustomizingExistingClasses.html#//apple_ref/doc/uid/TP40011210-CH6-SW2